libs/design/media-gallery/examples/src/basic-media-gallery/basic-media-gallery.component.html

```diff
@@ -0,0 +1,7 @@
+<daff-media-gallery>
```

Comment:
Currently, changing the active image of any image gallery will change the active image in every gallery to that image, even if that image is not in the list of images for that gallery. I tested this by adding another media-gallery here with fewer of the images.